src/neptune_scale/net/projects.py

```diff
@@ -82,3 +91,15 @@ def _safe_json(response: httpx.Response) -> Any:
         return response.json()
     except JSONDecodeError:
         return {}
+
+
+def get_project_list(*, api_token: Optional[str] = None) -> List[Dict]:
+    client = HostedApiClient(api_token=_get_api_token(api_token))
+
+    params = {
+        "userRelation": "viewerOrHigher",
+        "sortBy": "lastViewed",
+    }
+
+    response = client.backend.get_httpx_client().request("get", f"{PROJECTS_PATH_BASE}", params=params)
```

Comment:
```suggestion
    response = client.backend.get_httpx_client().request("get", PROJECTS_PATH_BASE, params=params)
```
is there a reason for f""? It's a string constant so no need to cast it right?